__main__.py

```diff
@@ -57,17 +60,24 @@ def main():
         elif args.command == 'citation':
             if args.dataset is None:
                 citation_path = os.path.join(os.path.split(__file__)[0], '../CITATION')
-                print citation_path
+                print "\ncitation for retriever\n"
                 with open(citation_path) as citation_file:
                     print citation_file.read()
             else:
                 scripts = name_matches(script_list, args.dataset)
                 for dataset in scripts:
-                    print dataset.description
+
+                    print "\ncitation::   %s" %dataset.citation 
+                    print "\ndescription::    %s\n" %dataset.description
```

Comment:
Capitalize 'citation' and 'description', and move to using `.format` syntax instead of `%`.